pages/post.js

```diff
@@ -158,23 +173,23 @@ const bodyOptions = {
 
       return (
         <p>
-          {children.reduce((acc, item) => {
-            if (typeof item === 'object' && item.type === 'a') {
-              const { props: { children: child, href } } = item;
-              const isDofollowLink = blogPostsDoFollowLinks.includes(href);
-              if (isDofollowLink) {
-                acc.push(<a href={href}>{child}</a>);
-              } else {
-                acc.push(<a rel="nofollow noopener" href={href}>{child}</a>);
-              }
-            } else if (item) {
-              acc.push(item);
-            }
-            return acc;
-          }, [])}
+          {renderNoFollowLinks(children)}
         </p>
       );
     },
+    [BLOCKS.HEADING_3]: (node, children) => {
+      const filteredChildren = children.filter((item) => !!item);
+
+      if (filteredChildren.length === 1 && typeof filteredChildren[0] === 'object') {
+        return filteredChildren[0];
```

Comment:
@stefan9541 , I believe you either need to add comments here, or use variables explaining meaning.